src/intents/schemas/lowLevel.ts

```diff
@@ -0,0 +1,117 @@
+import { segmentFunctionNames, signatureTypes, tokenStandards } from "./constants";
```

Comment:
we have been calling it "low level" definition, but we should come up with a better name for this.

Maybe we call it "functional"? so we would have DSL Definition level, and functional level